src/shared/logger/index.ts

```diff
@@ -0,0 +1,15 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// This file is the common import file, intended for use by most extension code.
+// It surfaces Logger and related interfaces, types, and retrieval.
```

Comment:
index.ts doesn't make a ton of sense as a name to me, thinking it should be renamed to types.ts or common.ts

Reply:
using `index.ts` allows you to do `import * as x from /src/shared/logger` (no filename) as a clean import point.